goodwe/es.py

```diff
@@ -141,10 +141,19 @@ class ES(Inverter):
         Integer("bp_pv_discharge", 38, "BP PV Discharge"),
         Integer("bp_bms_protocol", 40, "BP BMS Protocol"),
         Integer("power_factor", 42, "Power Factor"),
+        Integer("float_voltage", 44, "Float Voltage"),
```

Comment:
Float voltage should be type `Decimal` with precision 10 (it's expressed in units of 0.1V)